ci-conda.Dockerfile

```diff
@@ -37,7 +37,7 @@ EOF
 RUN <<EOF
 case "${LINUX_VER}" in
   "ubuntu"*)
-    apt-get update && apt-get install -y wget
+    rapids-retry apt-get update -y && apt-get install -y wget
```

Comment:
Package lists are failing to download. I think `APT::Acquire::Retries` only applies to files, not package lists. Retrying more aggressively, in the spirit of #285 / #288.